### PR TITLE
Fixing EXPLAIN command

### LINKED ISSUE #541: EXPLAIN support
**Is your feature request related to a problem? Please describe.**
Need to test EXPLAIN after the big upgrade.

### PR DESCRIPTION
**Pull request by subbyte**
Add tests for EXPLAIN in both cache sql and sqlalchemy interface. Fixed #541 

Fixing the following issues:
1. Construct non-serializable due to the use of DataFrame (not supported by `mashumaro`)
2. `deepcopy()` will return `DataFrame` instead of our subclass
3. Outdated `explain_graph()` in sqlalchemy interface

### CODE REVIEW

File: packages/kestrel_core/src/kestrel/ir/instructions.py [hunk 1]
```diff
@@ -33,6 +35,21 @@
     InitVar.__call__ = lambda *args: None
 
 
+class MashDataFrame(DataFrame, SerializableType):
```
Comment (pcoccoli) [inline packages/kestrel_core/src/kestrel/ir/instructions.py]: Not sure what `MashDataFrame` means.  If it's just a serializable DataFrame, maybe call it `SerializableDataFrame`?

File: packages/kestrel_core/src/kestrel/ir/graph.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 from __future__ import annotations
 
 import json
+import copy
```
Comment (pcoccoli) [inline packages/kestrel_core/src/kestrel/ir/graph.py:3]: We should try to keep the import sorted according to PEP 8.  A simple way to do this is using the `isort` tool (I find i works best *after* `unimport` but *before* `black`).

File: packages/kestrel_core/src/kestrel/ir/instructions.py (hunk 1 above)
Comment (subbyte) [reply packages/kestrel_core/src/kestrel/ir/instructions.py]: Good idea. Will rename

File: packages/kestrel_core/src/kestrel/ir/graph.py (hunk 2 above)
Comment (subbyte) [reply packages/kestrel_core/src/kestrel/ir/graph.py:3]: Thanks for the suggestion, will add to the github action.

**Review by pcoccoli — APPROVED** [review]

**Pull request merged**